backend/pom.xml

```diff
@@ -23,6 +23,7 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <java.version>1.8</java.version>
         <google-maps-services.version>0.2.7</google-maps-services.version>
+        <junit.version>LATEST</junit.version>
```

Comment:
I think that it's better to specify the version here instead of using LASTEST

Reply:
OK, i'll have to specify it for the four junit dependencies but it's much cleaner

Reply:
Yes I think that it's better this way 👍